### ISSUE TITLE
DayCycleSpeedScale

### ISSUE THREAD

**Issue by Xodus27** [thommcgrath/Beacon#317]
This is an active command in the ASA INI. I noticed Beacon doesn't add this as a configurable option.

**Comment by thommcgrath** [comment]
Correct. Beacon always sets it to 1.0, though importing does read the value and uses it to adjust the day and night scales accordingly. It keeps the math simpler.

**Comment by Xodus27** [comment]
This doesnt solve the problem. I am not looking to import it. Aberration ignores the day and night scales and because of this the only configurable line is the DayCycleSpeedScale that will affect Aberration. I used .002 to increase the entire day to my desired time but the only way I could do this was to change it in Nitrado directly.  I couldnt add it to Custom Config either to try and backend the code.  I know you are busy but this seems overlooked and dismissed since I have been bringing this up for close to a month.

**Comment by thommcgrath** [comment]
I didn't say it'd fix the problem. I'm saying it doesn't make sense to change Beacon's logic to adjust for Wildcard's bug, only to have to change it back later when the bug is fixed.

**Comment by Xodus27** [comment]
I am not sure that it's a bug. Relative to how the surface operates now the exact scheduling of nameless and Reapers on the surface.  This will not change no matter what the cycles are.  The actual day and night cycle is purely a lighted and dark aesthetic and doesn't affect the Nameless and Reaper schedule which is the most important aspect of the day and night cycle.  If you are suggesting to just wait until "The Bug" is fixed. How long is long enough to wait before we may start to assume it will not be changed? Can we at least allow the Custom Config to adjust the setting even if its not added to the rest of the project?

**Comment by thommcgrath** [comment]
The bug appears to be that the map is using the wrong day cycle manager. Aberration's day cycle manager is present in ASA, and still has the same day cycle information as ASE.

If you want to override it with custom config, just make sure you're not using Beacon's day cycle editor.
